components/brave_rewards/resources/ui/components/walletWrapper/index.tsx

```diff
@@ -156,6 +156,7 @@ export type Step = '' | 'captcha' | 'complete'
 
 interface State {
   verificationDetails: boolean
+  hideDisconnectWalletError: boolean
```

Comment:
Can this state (and the other changes to this file) be removed now?

Reply:
updated 👍